ipfs-cluster-service/main.go

```diff
@@ -276,7 +277,16 @@ removal, upgrade state using this command, and restart every peer.
 		if c.Bool("debug") {
 			setupDebug()
 		}
-		return nil
+
+		if _, err := os.Stat(absPath); err != nil {
```

Comment:
I think abspath is always going go be set here so you'll have a place to lock.

I would add a hidden --lockfile flag which pretty much gets the same treatment as the `config` flag (default initialization etc) but eventually allows to customize the lock location.